templateRunScripts/run.cctm

```diff
@@ -354,20 +353,20 @@ endif
  source $BASE/outck.q
 
  ls -l $BLD/$EXEC; size $BLD/$EXEC
- unlimit
- limit
+# unlimit
+# limit
 
 #> Executable call for single PE, uncomment to invoke
 if ( $PROC == 'serial' ) then
- /usr/bin/time  $BLD/$EXEC
+ time  $BLD/$EXEC
 else
 #> Executable call for multi PE, configure for your system 
  ## set MPIRUN = mpirun
  ## set MPIRUN = /mnt/appsource/intel_studio/intel_studio-2016.3/compilers_and_libraries_2016.3.210/linux/mpi/intel64/bin/mpirun
  set MPIRUN = mpirun
  ## time $MPIRUN -r ssh -np $NPROCS $BLD/$EXEC
  ## /usr/bin/time $MPIRUN -r ssh -np $NPROCS $BLD/$EXEC
- /usr/bin/time $MPIRUN -np $NPROCS $BLD/$EXEC
+ time $MPIRUN -np $NPROCS $BLD/$EXEC
```

Comment:
This is a material difference because `time` is usually a built-in of the shell which has a different output format than `/usr/bin/time` the program. I guess the output is not parsed anywhere, but if it is, using `/usr/bin/time` is reasonable.

Reply:
Good to know. It's not parsed anywhere and is purely informative. There are cases where `time` was used and some where `/usr/bin/time`. Turns out the basic docker container doesn't include /usr/bin/time program so I'm happy to coalesce on `time` as we don't parse the result